libs/agentc_core/agentc_core/util/ddl.py

```diff
@@ -108,18 +112,19 @@ def create_vector_index(
 
     max_partition_env = os.getenv("AGENT_CATALOG_MAX_SOURCE_PARTITION")
     try:
-        max_partition_env = int(max_partition_env)
+        max_partition = int(max_partition_env) if max_partition_env is not None else 1024
```

Comment:
ref to https://github.com/couchbaselabs/agent-catalog/pull/26/ for this - you can revert this change